Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -343,7 +346,7 @@ namespace O3DE::ProjectManager
             pybind11::finalize_interpreter();
         }
 
-        return !PyErr_Occurred();
```

Comment:
With python 3.10, once the python interpreter is shutdown, any calls, even ones to check the status of the interpreter, will produce a segfault.